content/en/tracing/profiling/getting_started.md

```diff
@@ -0,0 +1,229 @@
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_SERVICE`                                     | String        | The Datadog [service][3] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example `production`, which can be set here, or in `DD_PROFILING_TAGS` with `DD_PROFILING_TAGS="env:production"`. |
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.   |
+
+<div class="alert alert-info">
+Recommended for advanced usage only.
+</div>
+
+- If you want to manually control the lifecycle of the profiler, use the `ddtrace.profiling.profiler.Profiler` object:
+
+    ```python
+    from ddtrace.profiling import Profiler
+
+    prof = Profiler()
+    prof.start()
+
+    # At shutdown
+    prof.stop()
+    ```
+
+[1]: https://app.datadoghq.com/account/settings#agent/overview
+[2]: https://app.datadoghq.com/profiling
+[3]: /tracing/visualization/#services
+[4]: /tracing/guide/setting_primary_tags_to_scope/#environment
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+The Datadog Profiler requires Go 1.12+. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1] or [6.20.2][1].
+
+2. Get `dd-trace-go` using the command:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1
+    ```
+
+     **Note**: Profiling is available in the `dd-trace-go` library for versions 1.23.0+.
+
+3. Import the [profiler][2] at the start of your application:
+
+    ```Go
+    import "gopkg.in/DataDog/dd-trace-go.v1/profiler"
+    ```
+
+4. Add the following snippet to start the profiler:
+
+    ```Go
+    err := profiler.Start(
+        profiler.WithService("<SERVICE_NAME>"),
+        profiler.WithEnv("<ENVIRONMENT>"),
+        profiler.WithTags("version:<APPLICATION_VERSION>"),
+    )
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer profiler.Stop()
+    ```
+
+4. After a minute or two, visualize your profiles in the [Datadog APM > Profiling page][3].
+
+**Note**:
+
+- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use profiler configuration to set the parameters:
```

Comment:
I'd personally change this section on all languages to describe how to configure both:
* With environment variables
* With code (where available)

In this PR we have a mix between one version and the other, with some hints as to the existence of an alternative configuration. Lets be explicit imo.

Reply:
Yeah for sure, I think instead of making a full blown two paths example, ill simply add a note saying  `Alternatively you can also set the profiler configuration using environment variables:` with env var tables!

Reply:
This is also true for -D vs env var in java.

And same comment as java on language.